libs/shared/feature/form-nuv/src/lib/services/form-transaction.service.ts

```diff
@@ -102,6 +102,13 @@ export class FormTransactionService {
     );
   }
 
+  /**
+   * Set the form configuration manually when configuration is not tied to transaction (builder)
+   */
+  public setformConfiguration(formModel: FormModel) {
+    this._form.next(formModel);
+  }
```

Comment:
It would be simpler if you just tet the formconfiguration like this

```suggestion
  public setformConfiguration(formConfigurationModel: FormConfigurationModel) {
    this._form.next(
    this._form.value,
    formConfigurationModel,
    );
  }
```